src/Process/StateChangeCallback.php

```diff
@@ -3,9 +3,19 @@
 
 namespace PHPSu\Process;
 
-use Symfony\Component\Console\Output\ConsoleSectionOutput;
+use PHPSu\Tools\EnvironmentUtility;
 use Symfony\Component\Console\Output\OutputInterface;
 
+if (!class_exists('ConsoleSectionOutput', false)) {
+    if (version_compare((new EnvironmentUtility())->getSymfonyProcessVersion(), '4.0.0', '>=')) {
+        \class_alias(\Symfony\Component\Console\Output\ConsoleSectionOutput::class, 'ConsoleSectionOutput');
+    } else {
+        \class_alias(\PHPSu\Tools\ConsolePolyfill\ConsoleSectionOutput::class, 'ConsoleSectionOutput');
```

Comment:
maybe we could rename our class like the original. Than we could remove this if condition in every file. And only have this condition in the Polyfill File.

What do you think about this @ChrisB9?

Reply:
That sounds like a great idea and i saw your implementation and liked it so far!